odoo/addons/base/data/res_currency_data.xml

```diff
@@ -22,8 +22,9 @@
 
         <record id="CAD" model="res.currency">
             <field name="name">CAD</field>
-            <field name="symbol">$</field>
+            <field name="symbol">CA$</field>
```

Comment:
??? Not sure Canadians will agree that their dollars have the symbol CA$ now @lap-odoo

Reply:
@jva-odoo

Reply:
@rgo-odoo

Reply:
@jco-odoo reason behind this is only to remove confusion when multi-currency with $ sign

So as your comment I have other solutions that I see when finding thous symbols

The solution is we add new fields in currency model like an alternative_symbol in this field we add alternatively symbol like CA$ and when user use multi-currency at that time we use  alternatively symbol if available for all other currency

Note: other currency mins other than company currency.

So if Canadians like to use $ or any other country only like to use $ then it's possible with this solution

For example, If a Canadian company does the transaction in USD then the symbol is US$ so the confusion problem is also solved.